src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ClientDTO.java

```diff
@@ -9,6 +10,7 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
What does this line do?

Reply:
this @JsonIgnoreProperties(ignoreUnknown = true)
 is set to ignore data that is not there in this Entity, in the frontend we submit a DTO with more fields that are not there and we need them in the backend